src/equations/acoustic_perturbation_2d.jl

```diff
@@ -420,6 +420,16 @@ end
 end
 
 
+# Specialized `DissipationLocalLaxFriedrichs` to avoid spurious dissipation in the mean values
```

Comment:
This is only necessary because of the strategy to carry the mean values as additional equations with zero fluxes, correct?

Reply:
Correct.